modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/js/SiteNavigationMenuItemDOMHandler.js

```diff
@@ -86,7 +85,19 @@ const getId = function (menuItem) {
  * @return {HTMLElement|null}
  */
 const getNextSibling = function (menuItem) {
-	next(menuItem, `.${MENU_ITEM_CLASSNAME}`);
+	const selector = `.${MENU_ITEM_CLASSNAME}`;
+
+	let element = menuItem;
+
+	while (element) {
+		element = element.nextSibling;
+
+		if (element && element.matches(selector)) {
```

Comment:
If you want to check for the presence of a classname, it is probably better to use the more targeted API for it, and avoid the whole `selector` template literal.

```js
if (element && element.classList.has(MENU_ITEM_CLASSNAME))
```

We had [a semi-related discussion in the guidelines repo](https://github.com/liferay/liferay-frontend-guidelines/issues/19) about how best to select element and the consensus seemed to be that we should use more specific/targeted APIs when available, so I think the same applies here analogously.

Reply:
👍  good idea